Dockerfile_browser_tools.dockerfile

```diff
@@ -10,12 +10,12 @@ RUN sudo add-apt-repository --yes ppa:malteworld/ppa
 RUN sudo apt install pdftk --allow-unauthenticated
 
 # Install Puppeteer via Yarn
-RUN yarn add puppeteer@21.9.0
+RUN yarn add puppeteer@22.3.0
 
 # Install Chrome using Puppeteer command
-RUN npx puppeteer browsers install chrome@121
+RUN npx puppeteer browsers install chrome@122
```

Comment:
Do we have to specify the chrome version, can we just have:

`RUN npx puppeteer browsers install chrome`